gcivil-nyu-org/fall2019-cs-gy-6063-team-debug_entities
# PR for #153: Fix the UI and Bugs on /events/

### Description

* Changed the filters on the events page to use dropdowns to hopefully look better
* Fixed double-clicking on interested or going so that it works consistently

### Testing Directions

```
python manage.py makemigrations
python manage.py migrate
python manage.py pull_seatgeek_data
python manage.py make_users
```

1. Get the code from this PR, run the Django server, go to the events page
1. Click on the boroughs, performers, venues, and genres filters and see that they're now dropdowns instead of ugly lists
> ![dropdowns](https://user-images.githubusercontent.com/13837978/69116251-b566c600-0a59-11ea-8db4-e31b6abd99e8.gif)
1. Choose some filters, click submit, and see that the events match your criteria
1. Click "Interested" on some event, see that the button changes to "Not Interested", click that and see that it changes back to "Interested". Do the same on another event but with "Going" instead of "Interested".
> ![interested](https://user-images.githubusercontent.com/13837978/69116412-5ce3f880-0a5a-11ea-8997-699c18c6e43d.gif)


### Miscellaneous Comments

* When the page is loading, you can see the old ugly filters for a split second. I don't know why. I'm guessing it's because the Javascript takes a second to load.

### Checklist
